front-end/src/Auth.js

```diff
@@ -31,6 +31,10 @@ class Auth {
     return this.idToken;
   }
 
+  getAccessToken() {
```

Comment:
Are you calling this method anywhere? If not, I think it would be safe to delete it, since you are referencing `this.accessToken` directly later in your code